client/src/components/Instructions/index.js

```diff
@@ -266,6 +266,33 @@ export default class Instructions extends Component {
             Reload
           </Button>
         </div>
+        <h2> Upgrading on Development Network </h2>
```

Comment:
I would wrap all this section and only show it if it is local ganache. If it is, hide the button above and show these instructions

Reply:
otherwise show the normal ones